official/resnet/keras/resnet_model.py

```diff
@@ -218,28 +218,30 @@ def ResNet50(include_top=True,
   x = tf.keras.layers.Activation('relu')(x)
   x = tf.keras.layers.MaxPooling2D((3, 3), strides=(2, 2))(x)
 
-  x = conv_block(x, 3, [64, 64, 256], stage=2, block='a', strides=(1, 1))
+  strides = 2
+
+  x = conv_block(x, 3, [64, 64, 256], stage=2, block='a', strides=strides)
   x = identity_block(x, 3, [64, 64, 256], stage=2, block='b')
   x = identity_block(x, 3, [64, 64, 256], stage=2, block='c')
 
-  x = conv_block(x, 3, [128, 128, 512], stage=3, block='a')
+  x = conv_block(x, 3, [128, 128, 512], stage=3, block='a',strides=strides)
   x = identity_block(x, 3, [128, 128, 512], stage=3, block='b')
   x = identity_block(x, 3, [128, 128, 512], stage=3, block='c')
   x = identity_block(x, 3, [128, 128, 512], stage=3, block='d')
 
-  x = conv_block(x, 3, [256, 256, 1024], stage=4, block='a')
+  x = conv_block(x, 3, [256, 256, 1024], stage=4, block='a', strides = strides)
   x = identity_block(x, 3, [256, 256, 1024], stage=4, block='b')
   x = identity_block(x, 3, [256, 256, 1024], stage=4, block='c')
   x = identity_block(x, 3, [256, 256, 1024], stage=4, block='d')
   x = identity_block(x, 3, [256, 256, 1024], stage=4, block='e')
   x = identity_block(x, 3, [256, 256, 1024], stage=4, block='f')
 
-  x = conv_block(x, 3, [512, 512, 2048], stage=5, block='a')
+  x = conv_block(x, 3, [512, 512, 2048], stage=5, block='a', strides=strides)
   x = identity_block(x, 3, [512, 512, 2048], stage=5, block='b')
   x = identity_block(x, 3, [512, 512, 2048], stage=5, block='c')
 
   if include_top:
-    x = tf.keras.layers.AveragePooling2D((7, 7), name='avg_pool')(x)
+    x = tf.keras.layers.AveragePooling2D((2, 2), name='avg_pool')(x)
```

Comment:
I am not a pro at networks but when I changed other examples from V1 to V1.5 I only  changed the 3x3 and first 1x1 not the pooling.  I have not looked closely at those example and I am have limited familiarity with keras currently.  Can you provide some detail/background and where you see others doing this along with why the change from 7,7 to 2,2.  Thank you @seemuch

Reply:
You are right, we need to make a change inside the conv_block function. I am not sure if setting strides has any impact since the default value of strides is already (2X2).

Reply:
I think I misunderstood how the change should be made. Thanks to @anj-s 's comments below, I now have a better understanding, and will make a change accordingly. 
As far as the default strides of the conv_block, yes @anj-s you are right, it is (2*2) by default, but when I explicitly set it to 2, it somehow messes up with the polling size. I don't understand why that is. But it is not the right place to make the change anyway, I will ignore it for now.